### DISCUSSION TITLE
No puedo desplegar mi aplicacion

### DISCUSSION THREAD

**Discussion by LuiSxMendoza** [domcloud/domcloud#74] [General]
![Captura de pantalla_31-3-2024_15179_my domcloud co](https://github.com/domcloud/domcloud/assets/83250650/2db78aea-dbd2-4945-a28e-8a1b0be4cc8b)

Pueden ayudarme?

**Comment by nelsonreyesdeveloper** [comment]
me pasa lo mismo

**Comment by willnode** [comment]
This was because of a bad regular system upgrade. We've downgraded it. Please try again.
